docs/vsts-README.md

```diff
@@ -152,15 +173,21 @@ Allows you to promote an app previously updated to iTunes Connect to the App Sto
 
 ![Promote task](/images/promote-task-with-advanced.png)
 
-1. **Username and Password** or **Service Connection** - The credentials used to authenticate with the App Store. Credentials can be provided directly or configured via a service connection that can be referenced from the task (via the `Service Connection` authentication method).
+1. **Authentication method** - What type of credentials will be used to authenticate with the App Store. Credentials can be provided directly (using `App Store Connect Api Key` or `Username and Password` options) or configured via a service connection that can be referenced from the task (via the `Service Connection` authentication method).
+
+2. **App Store Connect API Key ID, Issuer ID and Key Content (base64-encoded)** - Available only if authentication method is `App Store Connect Api Key`. The API key data used to authenticate with the App Store. Key content has to be base64-encoded.
```

Comment:
Could it make sense to split it and describe every parameter?

Reply:
I think it makes sense to keep these string inputs together since they are pretty self-descriptive. I'll add the info about the api key in-house checkbox though